src/components/structures/TimelinePanel.tsx

```diff
@@ -731,7 +731,6 @@ class TimelinePanel extends React.Component<IProps, IState> {
         let shouldSendRR = true;
 
         const currentRREventId = this.getCurrentReadReceipt(true);
-        const currentRREventIndex = this.indexForEventId(currentRREventId);
```

Comment:
Looks like this was the only thing `indexForEventId ` was used for, although I am a bit worried this will just start sending lots of unnecessary read receipts. Could we disable this check just for threads?